math/src/main/scala/breeze/linalg/BitVector.scala

```diff
@@ -117,7 +117,7 @@ object BitVector extends BitVectorOps {
       for(i <- from.valuesIterator) fn.visit(i)
     }
 
-    def isTraversableAgain(from: BitVector): Boolean = true
+    override def isTraversableAgain(from: BitVector): Boolean = true
   }
```

Comment:
i don't know that style guide, but I'll object if anything offends me

On Fri, Apr 8, 2016 at 9:51 AM, Kenta Takagaki <notifications@github.com>
wrote:

> In math/src/main/scala/breeze/linalg/BitVector.scala
> <https://github.com/ktakagaki/breeze/pull/17#discussion_r59055289>:
>
> > @@ -117,7 +117,7 @@ object BitVector extends BitVectorOps {
> >        for(i <- from.valuesIterator) fn.visit(i)
> >      }
> >
> > -    def isTraversableAgain(from: BitVector): Boolean = true
> > +    override def isTraversableAgain(from: BitVector): Boolean = true
> >    }
>
> @dlwh <https://github.com/dlwh>
> I don't know that there's an official style guide for breeze, but I'm
> loosely refactoring off of the databricks guide as I read the code and
> learn it, is that OK?
>
> —
> You are receiving this because you were mentioned.
> Reply to this email directly or view it on GitHub
> <https://github.com/ktakagaki/breeze/pull/17/files/0ef0bf906e05475a253f9ef782e4f603688f9063#r59055289>
>